siliconcompiler/schema/schema_cfg.py

```diff
@@ -222,19 +222,7 @@ def schema_fpga(cfg):
     ''' FPGA configuration
     '''
 
-    scparam(cfg, ['fpga', 'arch'],
-            sctype='[file]',
-            copy=True,
-            shorthelp="FPGA: architecture file",
-            switch="-fpga_arch <file>",
-            example=["cli: -fpga_arch myfpga.xml",
-                     "api: chip.set('fpga', 'arch', 'myfpga.xml')"],
-            schelp=""" Architecture definition file for FPGA place and route
-            tool. For the VPR tool, the file is a required XML based description,
-            allowing targeting a large number of virtual and commercial
-            architectures. For most commercial tools, the fpga part name provides
-            enough information to enable compilation and the 'arch' parameter is
-            optional.""")
```

Comment:
is this just going away? ie. not needed?
seems like you need it along with the rr_graph thing?

Reply:
Architecture files are unique per fpga partname, so need to handle it accordingly.